microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -41,7 +41,7 @@ We are aware that Chromium has revised the timelines for Manifest V2 sunset. We
 | TBD - See note above | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge using Enterprise policies. |
 | TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policies. |
 
```

Comment:
even if my comment is right in specifying 2025 for Enterprise Policy, maybe the spirit of this PR is to improve article to specify June 2024 for other scenarios.  So maybe add a clear sentence to differentiate that scenario.

todo: see if live version of this article links at all to Chromium timeline page - first inspection gave negative results on that.